src/modules/rover_ackermann/RoverAckermann.cpp

```diff
@@ -94,19 +113,49 @@ void RoverAckermann::Run()
 		break;
 	}
 
-	hrt_abstime now = hrt_absolute_time();
+	// Timestamps
+	hrt_abstime timestamp_prev = _timestamp;
+	_timestamp = hrt_absolute_time();
+	const float dt = (_timestamp - timestamp_prev) * 1e-6f;
+
+	// Acceleration slew rate
+	if (_param_ra_max_accel.get() > FLT_EPSILON && _param_ra_max_speed.get() > FLT_EPSILON
+	    && _nav_state != vehicle_status_s::NAVIGATION_STATE_ACRO
```

Comment:
What's the need being excluding ACRO?

Reply:
Currently the only difference between manual and acro is that acro ignores slew rates.
The idea is that you can set up RA_MAX_ACCEL s.t. the rover doesn't wheely during acceleration.
But if you want to do that you can set it to ACRO without having to change the slew rate parameters to -1.

Reply:
Included in https://github.com/PX4/PX4-Autopilot/pull/23310/commits/6730d226787792ece02c3af60d36a6d54f33894c